scylla-cdc/src/log_reader.rs

```diff
@@ -13,6 +14,11 @@ use crate::consumer::ConsumerFactory;
 use crate::stream_generations::GenerationFetcher;
 use crate::stream_reader::StreamReader;
 
+const SECOND_IN_MILLIS: i64 = 1_000;
+const SLEEP_INTERVAL: i64 = SECOND_IN_MILLIS / 10;
+const WINDOW_SIZE: i64 = SECOND_IN_MILLIS / 10 * 3;
+const SAFETY_INTERVAL: i64 = SECOND_IN_MILLIS / 10;
```

Comment:
Let's prefix those constants with `DEFAULT_` to indicate that those are the default values for those parameters.